tests/client/test_v1.py

```diff
@@ -0,0 +1,24 @@
+"""Tests for Shaarli REST API v1 client"""
+from unittest import mock
+
+from shaarli_client.client import ShaarliV1Client
+
+SHAARLI_URL = 'http://domain.tld/shaarli/'
+SHAARLI_SECRET = 's3kr37!'
+
+
+def test_constructor():
+    """Instantiate a new client"""
+    ShaarliV1Client(SHAARLI_URL, SHAARLI_SECRET)
+
+
+@mock.patch('requests.request')
```

Comment:
So in that case

 * calling `requests.request()` actually does nothing network-wise? Correct?
 * Is this mock applied to the whole `test_v1.py`? Or just `test_get_info_uri()` ?
  * If it's applied through the whole test suite, and, say, we later need to call the _real_ `requests.request()`, how do we remove the mock?

Thanks for the valuable info